server/scripts/race.js

```diff
@@ -8,6 +8,9 @@ module.exports.getAveragePackCurrent = function(packetArray) {
 };
 
 module.exports.getSecondsRemainingUntilChargedOrDepleted = function(averagePackCurrent, packAmpHours) {
+    if (Math.abs(averagePackCurrent) == 0) {
```

Comment:
Not sure why `Math.abs` is required here, is this to somehow account for floating point errors or is it unneeded?

Reply:
U r right sir